### PR TITLE
[KYUUBI #2424] [Improvement] add Flink compile version and Trino client compile version to KyuubiServer Log

### LINKED ISSUE #2424: [Improvement] KyuubiServer‘s Log should print Flink compile version and Trino client compile version
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/incubator-kyuubi/issues?q=is%3Aissue) and found no similar issues.

### What would you like to be improved?

now it prints

Version: 1.6.0-SNAPSHOT, Revision: 9e486080ffad43245c84cda9674196f15a226bd8, Branch: master, Java: 1.8, Scala: 2.12, Spark: 3.2.1, Hadoop: 3.3.2, Hive: 2.3.9

should add Flink compile version and Trino client compile version

### How should we improve?

should add Flink compile version and Trino client compile version

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by jiaoqingbo**
### _Why are the changes needed?_

fix #2424 

### _How was this patch tested?_
- [x] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by jiaoqingbo** [comment]
cc @pan3793

File: build/kyuubi-build-info [hunk 1]
```diff
@@ -28,6 +28,8 @@ echo_build_properties() {
   echo kyuubi_spark_version="$5"
   echo kyuubi_hive_version="$6"
   echo kyuubi_hadoop_version="$7"
+  echo kyuubi_flink_version="$8"
+  echo kyuubi_trino_client_version="$9"
```
Comment (SteNicholas) [inline build/kyuubi-build-info:32]:
```suggestion
  echo kyuubi_trino_version="$9"
```

File: kyuubi-server/src/main/scala/org/apache/kyuubi/server/KyuubiServer.scala [hunk 2]
```diff
@@ -122,7 +122,8 @@ object KyuubiServer extends Logging {
     info(s"Version: $KYUUBI_VERSION, Revision: $REVISION, Branch: $BRANCH," +
       s" Java: $JAVA_COMPILE_VERSION, Scala: $SCALA_COMPILE_VERSION," +
       s" Spark: $SPARK_COMPILE_VERSION, Hadoop: $HADOOP_COMPILE_VERSION," +
-      s" Hive: $HIVE_COMPILE_VERSION")
+      s" Hive: $HIVE_COMPILE_VERSION, Flink: $FLINK_COMPILE_VERSION," +
+      s" Trino Client: $TRINO_CLIENT_COMPILE_VERSION")
```
Comment (SteNicholas) [inline kyuubi-server/src/main/scala/org/apache/kyuubi/server/KyuubiServer.scala:126]:
```suggestion
      s" Trino: $TRINO_COMPILE_VERSION")
```

Comment (SteNicholas) [inline kyuubi-server/src/main/scala/org/apache/kyuubi/server/KyuubiServer.scala:126]:
```suggestion
      s" Trino: $TRINO_COMPILE_VERSION")
```

Comment (SteNicholas) [inline kyuubi-server/src/main/scala/org/apache/kyuubi/server/KyuubiServer.scala:126]:
```suggestion
      s" Trino: $TRINO_COMPILE_VERSION")
```

File: kyuubi-common/src/main/scala/org/apache/kyuubi/package.scala [hunk 3]
```diff
@@ -61,6 +63,8 @@ package object kyuubi {
   val SPARK_COMPILE_VERSION: String = BuildInfo.spark_version
   val HIVE_COMPILE_VERSION: String = BuildInfo.hive_version
   val HADOOP_COMPILE_VERSION: String = BuildInfo.hadoop_version
+  val FLINK_COMPILE_VERSION: String = BuildInfo.flink_version
+  val TRINO_CLIENT_COMPILE_VERSION: String = BuildInfo.trino_client_version
```
Comment (SteNicholas) [inline kyuubi-common/src/main/scala/org/apache/kyuubi/package.scala:67]:
```suggestion
  val TRINO_COMPILE_VERSION: String = BuildInfo.trino_client_version
```

**Review by SteNicholas — CHANGES_REQUESTED** [review]

**Review by pan3793 — APPROVED** [review]

**Comment by pan3793** [comment]
Thanks, merging to master

**Pull request closed without merging**